test/tests/override-cmd/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -eo pipefail
+
+image="$1"
+
+# Test that we can override the CMD with echo
+hello="world-$RANDOM-$RANDOM"
+output="$(docker run --rm "$image" echo "Hello $hello")"
```

Comment:
Actually, I'm wrong. It does remove volumes: https://github.com/docker/docker/pull/15718/files